### DISCUSSION TITLE
I cannot enter my project

### DISCUSSION THREAD

**Discussion by ChristopherB-NucleaS** [AntlerVC/firetable#1165] [Support / Q&A] [answered]
I have an admin account, and I cant enter my projects, it appears a message that says I need to sign in with an admin account. 

![image](https://user-images.githubusercontent.com/122119202/225694078-f9cd8b61-a20f-487e-83eb-7f7d3062c0d3.png)

**Comment by BeeBombshell** [comment, marked as answer]
Hey @ChristopherB-NucleaS, could you check if you're using the correct account that has the ADMIN permissions?
Another reason for this is could be that the required Firestore Rules are not set correctly, check: https://docs.rowy.io/how-to/roles#write-firestore-security-rules-with-user-roles.
Also make sure you have your account set with the ADMIN role using the instructions on that page.
